modules/client-configuration/pages/autoinst-reinstall.adoc

```diff
@@ -0,0 +1,14 @@
+[[reinstalling-registered-systems]]
+= Reinstalling Registered Systems
+
+To reinstall in place an already registered client, you first have to define an autoinstallation distribution and an autoinstallation profile. For details, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].
```

Comment:
```suggestion
To reinstall a registered client in-place, you must define an autoinstallation distribution and an autoinstallation profile.
 For more information, see xref:client-configuration:autoinst-xxxxx.adoc[] and xref:client-configuration:autoinst-xxxxx.adoc[].
```

Reply:
Yeh. It's the curse of not being a native English speaker.

Reply:
However, "first" is needed as it's only the first step.